pkgs/kde/third-party/applet-window-buttons/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, extra-cmake-modules
+, kcoreaddons
+, kdeclarative
+, kdecoration
+, wrapQtAppsHook
+, libplasma
+}:
+
+stdenv.mkDerivation rec {
+  pname = "applet-window-buttons";
+  version = "0.13.0";
+
+  src = fetchFromGitHub {
+    owner = "moodyhunter";
+    repo = "applet-window-buttons6";
+    rev = "v${version}";
+    hash = "sha256-S7JcDPo4QDqi/RtvreFNoPKwTg14bgaFGsuGSDxs5nM=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+    wrapQtAppsHook
```

Comment:
Is wrapQtAppsHook actually necessary? Plasmoids should generally be just libraries.

Reply:
You are right.